frontend/public/components/utils/resource-link.jsx

```diff
@@ -63,3 +64,16 @@ export const ResourceLink = connectToModel(
   });
 
 ResourceLink.displayName = 'ResourceLink';
+
+
+const NodeLink_ = (props) => {
+  const {name, flags: { CAN_LIST_NODE: listNodeFlag }} = props;
+  if (!name) {
+    return <span>-</span>;
+  }
+  return listNodeFlag
+    ? <Link to={`/k8s/cluster/nodes/${name}`}>{name}</Link>
+    : <span>{name}</span>;
+};
```

Comment:
Yeah I will add the `<React.Fragment>`, thanks!

Reply:
Vs the `<span>`?  Hmm. I've tended towards `<React.Fragment>` as a bit of a hack fix for when you want to return `> 1` siblings without a wrapper.  Seems like a `<span>` is fine here, no?

Reply:
Unless the raw text node is preferrable.

Reply:
Looking at adding in the `ResourceIcon`, honestly I think the only reason that `<ResourceLink>` was not used was to avoid the BIG `NOD` icon... otherwise there doesn't seem to be a point in having a separate `<NodeLink>` component.

So, if @tlwu2013 is cool with the swap `N` and `NS`, This component can be deleted.

Reply:
Using the span creates unnecessary span elements in the markup that's generated, whereas React.Fragement doesn't. So I'd prefer the fragment to the span.

There's syntactic sugar where you can do `<>foo</>` without needing to type React.Fragment. I think it might cause problems for eslint, though. We should look at that.

Here I'd prefer the raw text node, though.

Reply:
> ... This component can be deleted.

I don't think we can... You still need to connectToFlags to check if you can list the nodes. I'd rather not connect all resource links since it has performance implications

Reply:
Right, sorry, spoke too soon.  My local changes are:

```JavaScript 
const NodeLink_ = (props) => {
  const {name, flags: { CAN_LIST_NODE: listNodeFlag }} = props;
  if (!name) {
    return <React.Fragement>-</React.Fragement>;
  }
  return listNodeFlag 
    ? <ResourceLink kind="Node" name={name} title={name} />
    : <React.Fragement>{name}</React.Fragement>;
}
```
So its simplified, not eliminated.